clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/mapper/AppToDeployDcosServerGroupDescriptionMapper.groovy

```diff
@@ -101,9 +101,11 @@ class AppToDeployDcosServerGroupDescriptionMapper {
           name = pd.name
           port = pd.port
           protocol = pd.protocol
-          labels = pd.labels
+          //labels = pd.labels
           loadBalanced = pd.labels?.keySet()?.any { it.startsWith('VIP') } ?: false
-          exposeToHost = false
+
+          // TODO exposeToHost not used by DeployDcosServerGroupDescriptionToAppMapper
+          // exposeToHost -> require ports?
```

Comment:
This is just something that slipped through the changes between dev and integration (so many rebases), will remove.